packages/augur-core/src/contracts/para/interfaces/IParaOICash.sol

```diff
@@ -6,6 +6,8 @@ import 'ROOT/libraries/token/IERC20.sol';
 
 
 contract IParaOICash is IERC20 {
+    function deposit(uint256 _amount) external returns (bool);
+    function withdraw(uint256 _amount) external returns (bool _alwaysTrue, uint256 _payout);
```

Comment:
why have _alwaysTrue?

Reply:
This was actually in before my changes. Someone modified the ParaOICash to do that and I asked @pgebheim about it and he said needed changes were made *shrug*

Reply:
This is needed for matic support, it came along with the matic patch you merged a while back robert